hack/common.sh

```diff
@@ -114,6 +114,11 @@ run_WMCO() {
       fi
   fi
 
+  # Add the "openshift.io/cluster-monitoring:"true"" label to the operator namespace to enable monitoring
+  if ! oc patch ns $WMCO_DEPLOY_NAMESPACE -p '{"metadata":{"labels":{"openshift.io/cluster-monitoring":"true"}}}'; then
```

Comment:
Will `openshift-windows-machine-config-operator` be a valid value for  `$WMCO_DEPLOY_NAMESPACE` ?

Reply:
yes since it is the operator namespace we are creating in the step above and using it here.

Reply:
So, just wondering; When to use the `$WMCO_DEPLOY_NAMESPACE` variable or the hard-coded value `openshift-windows-machine-config-operator`?

Refer to line #123, #135, etc
![image](https://user-images.githubusercontent.com/75598896/116271860-62b13380-a74e-11eb-8c48-4fdf5e55e2d6.png)

Reply:
that's a good point @jrvaldes, I would think we could have reused the variable declared [here](https://github.com/openshift/windows-machine-config-operator/blob/aff550fdba1d688d63d4d4f9df1773e3a3eb9e35/hack/common.sh#L5) when creating the ns as well. @sebsoto is there are reason to use the hard coded value or we can stick to reusing the ENV var in the lines pointed out?

Reply:
In addition,  about [line #112](https://github.com/openshift/windows-machine-config-operator/blob/aff550fdba1d688d63d4d4f9df1773e3a3eb9e35/hack/common.sh#L112), I feel the `WMCO_DEPLOY_NAMESPACE` value is getting ignored. 

![image](https://user-images.githubusercontent.com/75598896/116282679-0b649080-a759-11eb-9089-ef47cee50506.png)

Reply:
I want to call out that fixing the usage of `$WMCO_DEPLOY_NAMESPACE` consistently is outside the scope of the PR. @jrvaldes feel free to open a PR that fixes this and we can move this discussion over there.

Reply:
Agree

Reply:
> is there are reason to use the hard coded value or we can stick to reusing the var in the lines pointed out?

No, we should stick to using $WMCO_DEPLOY_NAMESPACE wherever possible